invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -48,35 +48,36 @@ def MChits_and_detsim_electron_simulation_params(nexusMC_filename):
 
 
 def test_generate_ionization_electrons(MChits_and_detsim_electron_simulation_params):
-    xs, ys, zs, energies,\
-    wi, fano_factor, lifetime, drift_velocity,\
-    transverse_diffusion, longitudinal_diffusion = MChits_and_detsim_electron_simulation_params
+    _, _, _, energies,\
+    wi, fano_factor, _, _,\
+    _, _ = MChits_and_detsim_electron_simulation_params
 
-    secondary_electrons = generate_ionization_electrons(wi, fano_factor, energies)
+    secondary_electrons = generate_ionization_electrons(energies, wi, fano_factor)
 
     assert len(secondary_electrons)==len(energies)
     assert np.all(secondary_electrons>=0)
 
 
 def test_drift_electrons(MChits_and_detsim_electron_simulation_params):
-    xs, ys, zs, energies,\
+    _, _, zs, energies,\
     wi, fano_factor, lifetime, drift_velocity,\
-    transverse_diffusion, longitudinal_diffusion = MChits_and_detsim_electron_simulation_params
+    _, _ = MChits_and_detsim_electron_simulation_params
 
-    secondary_electrons = generate_ionization_electrons(wi, fano_factor, energies)
-    electrons = drift_electrons(lifetime, drift_velocity, zs, secondary_electrons)
+    secondary_electrons = generate_ionization_electrons(energies, wi, fano_factor)
+    electrons = drift_electrons(zs, secondary_electrons, lifetime, drift_velocity)
```

Comment:
As I alluded elsewhere, all this ignoring and reordering is begging for named components.